VideoLocker/src/main/java/org/edx/mobile/base/BaseFragmentActivity.java

```diff
@@ -43,7 +43,7 @@
 
 import de.greenrobot.event.EventBus;
 
-public abstract class BaseFragmentActivity extends RoboAppCompatActivity
+public abstract class BaseFragmentActivity extends BaseActivity
```

Comment:
Since `BaseActivity` is also a `FragmentActivity`, this name is not really accurate anymore I guess. Maybe rename to `BaseAppActivity` or something similar?